tiatoolbox/wsicore/wsireader.py

```diff
@@ -1236,7 +1236,7 @@ def save_tiles(
 
             # convert to baseline reference frame
             bounds = start_w, start_h, end_w, end_h
-            baseline_bounds = tuple([bound * (2**level) for bound in bounds])
+            baseline_bounds = tuple([bound * (2 ** level) for bound in bounds])
```

Comment:
I think new version of black will fail this change.
```suggestion
            baseline_bounds = tuple([bound * (2**level) for bound in bounds])
```